### PR TITLE
Create subliminalStimulation

### PR DESCRIPTION
**Pull request by aeidi89**

### CODE REVIEW

**Comment by aeidi89** [comment]
#333

File: instances/technique/subliminalStimulation [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/technique/subliminalStimulation",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
+  "definition": "'subliminal stimulation' is a technique providing any sensory stimuli below an individual's threshold for conscious perception (adapted from [wikipedia](https://en.wikipedia.org/wiki/Subliminal_stimuli))",
```
Comment (lzehl) [inline instances/technique/subliminalStimulation]: "A 'si..."

**Review by lzehl — CHANGES_REQUESTED** [review]
please add the file extension (.jsonld)

File: instances/technique/subliminalStimulation (hunk 1 above)
Comment (aeidi89) [reply instances/technique/subliminalStimulation]: I have made the requested changes.

**Review by lzehl — APPROVED** [review]

**Pull request merged**